sentry_dart_plugin/README.md

```diff
@@ -0,0 +1,57 @@
+# Sentry Dart Plugin
+
+A Dart Build Plugin that uploads symbols for Android, iOS/macOS and Web to Sentry via [sentry-cli](https://docs.sentry.io/product/cli/).
+
+For doing it manually, please follow our [docs](https://docs.sentry.io/platforms/flutter/upload-debug/).
+
+## :clipboard: Install
+
+In your `pubspec.yaml`, add `sentry_dart_plugin` as a new dev dependency.
+
+```yaml
+dev_dependencies:
+  sentry_dart_plugin: ^1.0.0-alpha.1
+```
+
+## Run
+
+```bash
+dart run sentry_dart_plugin
```

Comment:
I think this should be `flutter pub run sentry_dart_plugin` if you're working on a Flutter app because Flutter uses it's own pub cache.
Though I'm not 100% sure on that.

So maybe add an example for Dart and one for Flutter here?

Reply:
both works in the same way, not sure about the pub caching thingie.
since this plugin is tooling agnostic, running on dart or flutter won't make any difference, obviously we can also mention that `flutter pub run` would also work